### PR TITLE
Relational Knowledge Distillation

### PR DESCRIPTION
**Pull request by khizirsiddiqui**
Passes ```epochs=1``` tests on local machine, so changed to ```epochs=0``` to prevent CI Tests from hanging.

Addresses issue #12

### CODE REVIEW

File: KD_Lib/original/mnist.py [hunk 1]
```diff
@@ -6,47 +6,51 @@
...
+from KD_Lib.RKD import RKDLoss
 
 
-def mnist(teacher_size=1200, student_size=800,epochs=20, lr=0.01, optimizer='SGD',
-          loss='MSE', batch_size=100, distil_weight=0.7, temp=20 ):
-    
+def mnist(teacher_size=1200, student_size=800, epochs=20, lr=0.01,
+          optimizer='SGD', loss='MSE', batch_size=100, distil_weight=0.7,
+          temp=20, rkd_angle=0.4, rkd_dist=0.6):
+
```
Comment (Het-Shah) [inline KD_Lib/original/mnist.py:15]: add doc string here so people know about various options

Comment (khizirsiddiqui) [reply KD_Lib/original/mnist.py:15]: We could do that now or later when we formalize it into proper functions? If you still suggest, I will.

Comment (Het-Shah) [reply KD_Lib/original/mnist.py:15]: yea do it now only, we will update it along the way

Comment (khizirsiddiqui) [reply KD_Lib/original/mnist.py:39]: :+1:

**Comment by Het-Shah** [comment]
Great will merge after this build finishes.

**Pull request merged**